packages/apps/docs/.vitepress/config.ts

```diff
@@ -0,0 +1,84 @@
+import { defineConfig } from 'vitepress';
+import { addContextLinkPlugin } from './plugin';
+
+export default defineConfig({
+  title: 'Fuels-ts',
+  srcDir: 'src',
+  markdown: {
+    config: (md) => {
+      md.use(addContextLinkPlugin);
```

Comment:
Amazing. 👏 

One of our simplest yet best doc features is the `See Code in Context` link we currently show after every imported code snippet. I can't repeat enough how useful I find that to be. The fact no one has implemented it in any tool only shows how accurate @camsjams was with that Typedoc plugin, and I'm sure this will be the final touch for all other snippets goodies Vitepress provides.

Good job keeping it alive via this `addContextLinkPlugin`!